code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -271,7 +271,9 @@ Judgement: 12 servants, 5 caches, 300 CV, and any existing AIs are converted or
 	return ..()
 
 /datum/clockwork_scripture/ranged_ability/scripture_effects()
-	slab.icon_state = slab_icon
+	if(slab_overlay)
+		slab.add_overlay(slab_overlay)
+		slab.item_state = slab_overlay
```

Comment:
This has also been fixed, in case you weren't aware due to how Git handles line comments.